src/Scripting/CoreTest/ScriptOptionsTests.cs

```diff
@@ -167,6 +167,7 @@ public void WithEmitDebugInformation_SameValueTwice_DoesNotCreateNewInstance()
         }
 
         [Fact]
+
         public void WithFileEncoding_SetsWithFileEncoding()
```

Comment:
Extra new line?

Reply:
Whoops, good catch - shouldn't be there.